src/app/models/Product.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface Product {
+  uid: number; // This is an auto-generated field but we are including this for lisibility sake, will be set to the id of the document
```

Comment:
```suggestion
  id: string; // This is an auto-generated field but we are including this for lisibility sake, will be set to the id of the document
```
`uid` generally stand for user identifiant 
In Firebase id are string sorry bro :smile: